sdk/src/main/java/com/sygic/travel/sdk/di/placesModule.kt

```diff
@@ -0,0 +1,61 @@
+package com.sygic.travel.sdk.di
+
+import com.github.salomonbrys.kodein.Kodein
+import com.github.salomonbrys.kodein.bind
+import com.github.salomonbrys.kodein.instance
+import com.github.salomonbrys.kodein.singleton
+import com.sygic.travel.sdk.common.StApiConstants
+import com.sygic.travel.sdk.common.interceptors.HeadersInterceptor
+import com.sygic.travel.sdk.common.interceptors.LocaleInterceptor
+import com.sygic.travel.sdk.places.api.PlacesApiClient
+import com.sygic.travel.sdk.places.facade.PlacesFacade
+import com.sygic.travel.sdk.places.service.PlacesService
+import com.sygic.travel.sdk.utils.UserAgentUtil
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.util.concurrent.TimeUnit
+
+
+internal val placesModule = Kodein.Module {
+	bind<HeadersInterceptor>() with singleton {
+		HeadersInterceptor(
+			apiKey = instance("apiKey"),
+			userAgent = UserAgentUtil.createUserAgent(instance())
+		)
+	}
+
+	bind<LocaleInterceptor>() with singleton { LocaleInterceptor() }
+
+	bind<OkHttpClient>("placesHttpClient") with singleton {
+		val builder = OkHttpClient.Builder()
+
+		if (instance("isInDebugMode")) {
+			builder.addInterceptor(instance<HttpLoggingInterceptor>())
+		}
+
+		builder.addInterceptor(instance<HeadersInterceptor>())
+			.addInterceptor(instance<LocaleInterceptor>())
+			.cache(instance())
+			.readTimeout(60, TimeUnit.SECONDS)
+			.build()
+	}
+
+	bind<Retrofit>("placesRetrofit") with singleton {
+		Retrofit.Builder()
+			.client(instance("placesHttpClient"))
+			.baseUrl("https://api.sygictravelapi.com/${StApiConstants.VERSION_AND_LOCALE}/")
```

Comment:
- please inline the constant

Reply:
In the project, there are 3 occurrences of this constant. Should I inline remaining 2 occurrences ?

Reply:
ok, let it be :)